server/src/main/java/org/apache/druid/guice/http/LifecycleUtils.java

```diff
@@ -26,7 +26,7 @@
 {
   public static Lifecycle asMmxLifecycle(Lifecycle lifecycle)
   {
-    final Lifecycle metamxLifecycle = new Lifecycle();
+    final Lifecycle metamxLifecycle = new Lifecycle("http-client");
```

Comment:
We could probably look into merging this lifecycle into the top level module lifecycle since i think historically these were different concrete types, but I think I would rather do that in a different PR.